test/models/user_test.rb

```diff
@@ -3,12 +3,13 @@
 class UserTest < ActiveSupport::TestCase
 
   def setup
-    @user = User.create(username: 'user',
+    @user = User.new(username: 'user',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.